src/patternfly/components/Menu/menu.scss

```diff
@@ -0,0 +1,253 @@
+
+  // select icon
+  --pf-c-menu__item-select-icon--MarginLeft: var(--pf-global--spacer--sm);
+  --pf-c-menu__item-select-icon--Color: var(--pf-global--active-color--100);
+  --pf-c-menu__item-select-icon--FontSize: var(--pf-global--icon--FontSize--sm);
+
+  // external icon
+  --pf-c-menu__item-main__external-icon--MarginLeft: var(--pf-global--spacer--sm);
+  --pf-c-menu__item-main__external-icon--Color: var(--pf-global--link--Color);
+  --pf-c-menu__item-main__external-icon--FontSize: var(--pf-global--icon--FontSize--sm);
+
+  // action
+  --pf-c-menu__item-action--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-menu__item-action--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-menu__item-action--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-menu__item-action--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-menu__item-action-icon--Color: var(--pf-global--Color--dark-200);
+  --pf-c-menu__item-action-icon--Height: calc(var(--pf-c-menu__item--FontSize) * var(--pf-c-menu__item--LineHeight));
+  --pf-c-menu__item-action--hover__icon--Color: var(--pf-global--Color--dark-100);
+  --pf-c-menu__item-action--m-favorite__icon--Color: var(--pf-global--disabled-color--200);
+  --pf-c-menu__item-action--m-favorite__icon--FontSize: var(--pf-global--icon--FontSize--sm);
+  --pf-c-menu__item-action--m-favorite--m-favorited__icon--Color: var(--pf-global--palette--gold-400);
+
+  padding-top: var(--pf-c-menu--PaddingTop);
+  padding-bottom: var(--pf-c-menu--PaddingBottom);
+  background-color: var(--pf-c-menu--BackgroundColor);
+  box-shadow: var(--pf-c-menu--BoxShadow);
+
+  &.pf-m-flyout {
+    .pf-c-menu {
+      position: absolute;
+      top: var(--pf-c-menu--m-flyout__menu--Top);
+      left: 100%;
+    }
+  }
+
+  .pf-c-divider {
+    margin-top: var(--pf-c-menu--c-divider--MarginTop);
+    margin-bottom: var(--pf-c-menu--c-divider--MarginBottom);
+  }
+}
+
+// Header
+.pf-c-menu__header {
+  padding-top: var(--pf-c-menu__header--PaddingTop);
+  padding-right: var(--pf-c-menu__header--PaddingRight);
+  padding-bottom: var(--pf-c-menu__header--PaddingBottom);
+  padding-left: var(--pf-c-menu__header--PaddingLeft);
+}
+
+// List item
+.pf-c-menu__list-item {
+  position: relative;
+  display: flex;
+  color: var(--pf-c-menu__list-item--Color);
+  background-color: var(--pf-c-menu__list-item--BackgroundColor);
+
+  &:hover:not(.pf-m-disabled),
+  &:focus-within:not(.pf-m-disabled) {
+    --pf-c-menu__list-item--Color: var(--pf-c-menu__list-item--hover--Color);
+    --pf-c-menu__list-item--BackgroundColor: var(--pf-c-menu__list-item--hover--BackgroundColor);
+
+    .pf-c-menu__item-external-icon {
+      opacity: 1;
+    }
+  }
+
+  &.pf-m-disabled {
+    .pf-c-menu__item {
+      --pf-c-menu__item--Color: var(--pf-c-menu__item--disabled--Color);
+
+      pointer-events: none;
+    }
+  }
+}
+
+// Item
+.pf-c-menu__item {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
```

Comment:
```suggestion
  flex-basis: 100%;
  min-width: 0;
```

`width: 100%` here causes this:

![Screen Shot 2020-08-20 at 2 00 03 PM](https://user-images.githubusercontent.com/5385435/90808333-d791bb00-e2ed-11ea-8e2c-85e7eb2b132b.png)